src/makefiles/cuda_32bit.mk

```diff
@@ -7,7 +7,8 @@ endif
 
 CUDA_INCLUDE= -I$(CUDATKDIR)/include
 CUDA_FLAGS = -g -Xcompiler -fPIC --verbose --machine 32 -DHAVE_CUDA \
-             -ccbin $(CXX) -DKALDI_DOUBLEPRECISION=$(DOUBLE_PRECISION)
```

Comment:
Does this still work? I notice that it doesn't specify an include directory for CUB. Also,  why don't we also define CUDA_API_PER_THREAD_DEFAULT_STREAM in cuda_64bit.mk?

Reply:
done